torch/_inductor/lowering.py

```diff
@@ -4520,6 +4520,11 @@ def fn(*args, **kwargs):
 register_inplace(aten.__ixor__, aten.__xor__)
 
 
+@register_lowering(aten.sym_constrain_range)
+def sym_constrain_range(a, min, max):
```

Comment:
should we assert that `a` is either an int or if its a symint that it's found in the tracing context ?

Reply:
Maybe I misunderstand, but at least in my testing `a` is neither an int nor a symint. It's a sympy Symbol. But I can check that it's in the tracing context.